includes/admin/tools/views/html-admin-page-system-info.php

```diff
@@ -693,7 +693,9 @@ class="dashicons dashicons-external"></span></a>
 		<td>
 			<?php
 			$last_paypal_ipn_received = get_option( 'give_last_paypal_ipn_received' );
-			$donation_id              = $last_paypal_ipn_received['payment_id'];
+			if ( is_array( $last_paypal_ipn_received ) ) {
+				$donation_id	 = $last_paypal_ipn_received['payment_id'];
+			}			
```

Comment:
Since `$donation_id` isn't otherwise defined and we use it below in the conditional, we should assign a null value.
```suggestion
			$donation_id = isset($last_paypal_ipn_received['payment_id']) ? $last_paypal_ipn_received['payment_id'] : null;		
```

Also, below in the condition, we should do:
```php
&& $donation_id !== null
&& get_post( $donation_id ) instanceof WP_Post
```
Otherwise, passing a null value to `get_post` causes it to attempt to get it from the request, which is _probably_ not a big deal in this context but is unnecessary.

Reply:
@coder-karen can you implement those suggestions in `if` condition?